tasks/engine_setup.yml

```diff
@@ -0,0 +1,61 @@
+---
+- block:
+  - name: Use the default answerfile
+    template:
+      src: answerfile_{{ ovirt_engine_setup_version }}_basic.txt.j2
+      dest: /tmp/answerfile.txt
+      mode: 0600
+      owner: root
+      group: root
+    when: ovirt_engine_setup_answer_file_path is undefined
+
+  - name: Copy custom answer file
+    no_log: "True"
+    template:
+      src: "{{ ovirt_engine_setup_answer_file_path }}"
+      dest: /tmp/answerfile.txt
+      mode: 0600
+      owner: root
+      group: root
+    when: ovirt_engine_setup_answer_file_path is defined
+
+  - name: Update setup packages
+    shell: 'yum -y update ovirt*setup*'
+    when: ovirt_engine_setup_update_setup_packages
+    tags:
+      - skip_ansible_lint
+
+  - name: Update all packages
+    shell: 'yum -y update'
```

Comment:
Why not use yum module?

Reply:
Same ^.
I 'll change and rerun tests I said. Maybe there was non related issue.

Reply:
Fixed.